lib/config/index.js

```diff
@@ -1327,7 +1327,6 @@ Config.prototype._canonicalize = function _canonicalize() {
   }
 
   this.serverless_mode.enabled = this.serverless_mode.enabled
```

Comment:
Seems like we can get rid of this line alltogether?

Reply:
doh, there should be a linting rule that says the left and right are the same